common/config/env/.env

```diff
@@ -1,2 +1,2 @@
 // THIS IS AN AUTO GENERATED FILE BY `RUSH SWITCH <BETA|STABLE>`. DO NOT MODIFY
-COMMUNICATION_REACT_FLAVOR=beta
\ No newline at end of file
+COMMUNICATION_REACT_FLAVOR=stable
```

Comment:
Remove. We clearly need to .gitignore this, or at leaset add a git-hook warning.